src/irmin-pack/inode_intf.ml

```diff
@@ -60,6 +80,9 @@ module type S = sig
        and type Portable.hash := hash
 
   val decode_bin_length : string -> int -> int
+  val snapshot_iter : value -> Val.Snapshot.t
```

Comment:
By the look of the signature it doesn't seem like a classic `iter` function. Wouldn't `to_snapshot` be a better name for it?